sql/moz-fx-data-shared-prod/monitoring_derived/jobs_by_organization_v1/query.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""
+
+Put data from the projects below into a table.
+
+  mozdata.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION,
+  moz-fx-data-shared-prod.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION,
+  moz-fx-data-marketing-prod.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION,
+  moz-fx-data-bq-data-science.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION
+into a table so that can be queried
+"""
+
+from argparse import ArgumentParser
+
+from google.cloud import bigquery
+
+DEFAULT_PROJECTS = [
+    "mozdata",
+    "moz-fx-data-shared-prod",
+    "moz-fx-data-marketing-prod",
+    "moz-fx-data-bq-data-science",
+]
+
+parser = ArgumentParser(description=__doc__)
+parser.add_argument("--date", required=True)  # expect string with format yyyy-mm-dd
+parser.add_argument("--project", default="moz-fx-data-shared-prod")
+parser.add_argument("--destination_project", default="moz-fx-data-shared-prod")
+parser.add_argument("--destination_dataset", default="monitoring_derived")
+parser.add_argument("--destination_table", default="jobs_by_organization")
+
+
+def create_query(date, project):
+    """Create query with filter for source projects."""
+    return f"""
+        SELECT
+          cache_hit,
+          creation_time,
+          destination_table,
+          end_time,
+          error_result,
+          job_id,
+          job_type,
+          parent_job_id,
+          principal_subject,
+          priority,
+          project_id,
+          project_number,
+          query,
+          referenced_tables,
+          reservation_id,
+          session_info,
+          start_time,
+          state,
+          statement_type,
+          timeline,
+          total_bytes_billed,
+          total_bytes_processed,
+          total_modified_partitions,
+          total_slot_ms ,
+          transaction_id,
+          user_email,
+          query_info.resource_warning ,
+          query_info.query_hashes.normalized_literals,
+          query_info.performance_insights,
+          transferred_bytes,
+          materialized_view_statistics,
+          DATE(creation_time) as creation_date
+        FROM
+          `{project}.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION`
+        WHERE
+          DATE(creation_time) = '{date}'
+          AND (t1.project_id IN UNNEST({DEFAULT_PROJECTS})
```

Comment:
It appears the `bigquery_usage_v2` table uses:
```sql
        FROM
          `{project}.region-us.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION` AS t1
        LEFT JOIN
          UNNEST(referenced_tables) AS referenced_tables
```

which is why `t1` exists in that query but not here. If I were writing this I would probably write something like:
```sql
AND (project_id IN UNNEST({DEFAULT_PROJECTS})
OR EXISTS(SELECT * FROM UNNEST(referenced_tables) AS x WHERE x.project_id IN UNNEST({DEFAULT_PROJECTS}))
OR destination_table.project_id IN UNNEST({DEFAULT_PROJECTS}))
 ```
to match the upstream `JOBS_BY_ORGANIZATION` as closely as possible, but I'm not a SQL expert.

Reply:
I've updated the code to what you have written with a change in the reference table name. Thinking if there is a clearer way to write it